compiler/onecc-docker/CMakeLists.txt

```diff
@@ -0,0 +1 @@
+add_subdirectory(tests)
```

Comment:
please add check if test is enabled.

from @seanshpark https://github.com/Samsung/ONE/pull/9793#discussion_r979705856